index.html

```diff
@@ -2033,15 +2033,24 @@ <h4>Presentations Using Derived Credentials</h4>
         <section>
           <h4>Presentations Including Holder Claims</h4>
           <p>
-A <a>holder</a> MAY use the <code>verifiableCredential</code>
-<a>property</a> in a <a>verifiable presentation</a> to include
-<a>verifiable credentials</a> from any <a>issuer</a>, including
-themselves. When the <a>issuer</a> of a <a>verifiable credential</a>
-is the <a>holder</a>, its <a>claims</a> are considered to be
-self-asserted. These self-asserted claims can be secured by their
-own mechanism like any other <a>verifiable credential</a> or by
-the same mechanism that secures the <a>verifiable presentation</a>
-in which they are included.
+A <a>holder</a> MAY use the <code>verifiableCredential</code> <a>property</a> in
+a <a>verifiable presentation</a> to include <a>verifiable credentials</a> from
+any <a>issuer</a>, including themselves. When the <a>issuer</a> of a
+<a>verifiable credential</a> is the <a>holder</a>, its <a>claims</a> are
+considered to be self-asserted. These self-asserted claims can be secured by
+their own mechanism like any other <a>verifiable credential</a> or by the same
+mechanism that secures the <a>verifiable presentation</a> in which they are
+included.
+          </p>
+          <p>
+The <a>subject</a> of these self-asserted <a>claims</a> is not limited and can
+include statements about the <a>holder</a>, one of the other included
+<a>verifiable credentials</a>, or even the <a>verifiable presentation</a> in
+which the self-asserted <a>verifiable credential</a> is included. In each of
+these cases the <code>CredentialSubject</code> would have an <code>id</code>
+<a>property</a> that matches the <code>id</code> of the <a>subject</a>. This is
+no different than how it is done in <a>verifiable credentials</a> that are not
+self-asserted.
```

Comment:
```suggestion
The <a>subject(s)</a> of these self-asserted <a>claims</a> are not limited, so
these <a>claims</a> can include statements about the <a>holder</a>, one of the
other included <a>verifiable credentials</a>, or even the <a>verifiable
presentation</a> in which the self-asserted <a>verifiable credential</a> is
included. In each of these cases, the <code>CredentialSubject</code> would
have an <code>id</code> <a>property</a> that matches the <code>id</code> of
the <a>subject</a>, just as it is done in <a>verifiable credentials</a> that
are not self-asserted.
```

Reply:
On top of @TallTed's edits, I'd think we'd just want to say that the subjects (little 's') in the VCs would have an `id` property instead of saying it only applies to the top-level credential subjects. Any subject will do in any claim made throughout the VC.